app/presenters/publishing_api_presenters/ministerial_role.rb

```diff
@@ -0,0 +1,29 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::MinisterialRole < PublishingApiPresenters::Item
```

Comment:
Completely understand the concern around duplication. This is very much an intermediate state that we're looking to resolve in our next PR. We've refrained from extracting out this duplication thus far so we can get a better handle on the kinds of repetition across all of the presenters. I think this'll let us make a more informed choice about the correct abstraction to use when sharing code across the classes, whether that's mixins or a small set of composable objects.